appconfigservice/api/controllers/app.py

```diff
@@ -43,6 +44,17 @@ def configs_ok_search():
     return msg
 
 
+def configs_version_search():
+    """
+    Method to return AppConfig BB version number
+    Reads yaml file and returns version number
+    Return : plain text - version number
+    """
+    appconfig_yaml = open("appconfig.yaml")
```

Comment:
@minump, I just realized why this was working when running using PyCharm. With PyCharm, I think it looks at the repository and tries to find this file. 

Please run the container using Docker. There is an error that it's not able to find the file. 

```
File "/app/appconfigservice/api/controllers/app.py", line 53, in configs_version_search
    appconfig_yaml = open("appconfig.yaml")
FileNotFoundError: [Errno 2] No such file or directory: 'appconfig.yaml'
```
I think this goes back to why it's important to run modules using Docker when reviewing code (note to myself). Please fix this. Thanks.

Reply:
one option to look at is:
```
from flask import current_app

def function():
  with app.app.app_context():
    current_app.config['env'] = 'dev'
```

see https://github.com/zalando/connexion/issues/119

Reply:
Hi, Rob,

I probably did not understand what you exactly meant. Could you please share a little more details? Just to clarify, this module is called App Config, and this PR was not about this module's config.